### DISCUSSION TITLE
installing Nuxt3-rc6 introduces a warning

### DISCUSSION THREAD

**Discussion by AliQ80** [nuxt/nuxt#17082] [Questions]
hello,

Installing nuxt 3 introduces the following warning 

warning vscode-languageclient@7.0.0: The engine "vscode" appears to be invalid.

what does it mean? and can I do something to fix it?

thank you

**Comment by jhisham** [comment]
hello, 

probably this link might help answer your question more effectively:

[engine 'vscode' appears to be invalid](https://github.com/microsoft/vscode/issues/91009)

**Reply by AliQ80** [reply]
thank you for your answer,

I have seen the issue you referenced, and the solution there is to add "--ignore-engines" to yarn install, but is ignoring the engine really the only solution, it would be helpful if we can figure out the cause of the problem, that's what I'm thinking anyway, especially since I have no idea what this engine is, or what are the implications if any to ignoring it.

at first I thought this maybe a vscode issue, but even when deploying to Vercel I see the same warning appear so it looks like it might be something else (I could be wrong though)
